pontoon/sync/core.py

```diff
@@ -20,6 +21,27 @@
 
 log = logging.getLogger(__name__)
 
+def run_only_one(func):
+    """
+    Decorator ensures that there's only one running task with given task_name.
+    """
+
+    task_name = "celery.{}".format(func.__name__)
+    def wrapped_func(*args, **kwargs):
+        # Moved import here to mock cache during tests.
+        from django.core.cache import cache
+
+        # Acquire the lock
+        if not cache.add(task_name, True):
+            return
```

Comment:
At a minimum we want to log here, and ideally we want to get a more noisy notification when this happens, as it's a bad thing if it does. #251 adds an error handler to tasks that takes exceptions raised in tasks and sends them to RayGun, so I think raising a `RuntimeError` with a descriptive message would be a good thing to do here.